spec/features/bulk_tags_edit_spec.rb

```diff
@@ -57,7 +57,7 @@
           wait_for_download
           druids_with_tags = CSV.parse(File.read(download))
           expect(druids_with_tags.count).to eq(number_of_druids)
-          druids_with_tags.each do |druid, tags|
+          druids_with_tags.each do |druid, *tags|
```

Comment:
the rubocop exception comment needs to be restored too, i think: https://github.com/sul-dlss/infrastructure-integration-test/pull/666/commits/c4edb8e7123ab38cbeab016d5f28ef37b470907c#diff-1413f9d516eee302bb7e789fe56d0135bf7edc823b4dfbf04b63c5ece6188511L60

```suggestion
          druids_with_tags.each do |druid, *tags| # rubocop:disable Style/HashEachMethods rubocop seems mistaken in thinking the tags var is unused
```